src/patternfly/components/Page/page.scss

```diff
@@ -287,6 +297,29 @@
     }
   }
 
+  .pf-c-button.pf-m-opened {
```

Comment:
`.pf-m-opened` is distinct to something being `open` in the UI, I think we can abstract to use the active state `.pf-m-active` here instead.

Reply:
I do agree with your comment about open being too specific as a name. But I'm wondering if `pf-m-active` would be confused with the `:active` selector? 

@mcarrano @maryshak1996 any thoughts here?

Reply:
I don't really have an opinion @christiemolloy . Maybe a better question for devs?